conf/log4j.properties.template

```diff
@@ -28,8 +28,8 @@ log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}:
 log4j.logger.org.apache.spark.repl.Main=WARN
 
 # Settings to quiet third party logs that are too verbose
-log4j.logger.org.spark_project.jetty=WARN
-log4j.logger.org.spark_project.jetty.util.component.AbstractLifeCycle=ERROR
+log4j.logger.org.spark.project.jetty=WARN
```

Comment:
None of these other files should be changed.

Reply:
Can you elaborate? `spark.shade.packageName` parent parameter is used across the entire project. 
Are you suggesting to have different shade prefix for `common/network-yarn` module only?